releasenotes/releasenotes_v5.rst

```diff
@@ -5,8 +5,129 @@ Current Firely Server release notes (v5.x)
 
 .. _vonk_releasenotes_5_0_0:
 
-Release 5.0.0-beta1, [TBD: specify date]
-----------------------------------------
+Release 5.0.0, February 9th, 2023
+----------------------------------
+
```

Comment:
We should mention that this release is now certified for "§170.315 (g)(10) Standardized API for patient and population services, see https://fire.ly/g10-certification/.